Doc/library/warnings.rst

```diff
@@ -522,6 +522,53 @@ Available Functions
    and calls to :func:`simplefilter`.
 
 
+.. decorator:: deprecated(msg, *, category=DeprecationWarning, stacklevel=1)
+
+   Decorator to indicate that a class, function or overload is deprecated.
+
+   When this decorator is applied to an object,
+   :term:`static type checkers <static type checker>`
+   will generate a diagnostic on usage of the deprecated object.
```

Comment:
```suggestion
   When this decorator is applied to an object,
   deprecation warnings will be emitted at runtime when the object is used.
   :term:`static type checkers <static type checker>`
   will also generate a diagnostic on usage of the deprecated object.
```

Reply:
Sure, but changing the first "will" into "may" because there are various circumstances where that would not happen.